src/main/java/xyz/nucleoid/plasmid/game/ManagedGameSpace.java

```diff
@@ -174,7 +174,7 @@ public void openGame(Consumer<GameLogic> builder) {
             closedGameLogic.getResources().close();
 
             try {
-                closedGameLogic.getListeners().invoker(GameCloseListener.EVENT).onClose();
+                closedGameLogic.getListeners().invoker(GameCloseListener.EVENT).onClose(null);
```

Comment:
Maybe this should have a specific close reason type for switching?

Reply:
Fixed in 1ad427eb96fbed995728f81c7e712224efe0f68c